sdk/communication/azure-communication-phonenumbers/README.md

```diff
@@ -30,7 +30,7 @@ Purchased phone numbers can come with many capabilities, depending on the countr
 
 ### Initializing Phone Number Client
 The PhoneNumberClientBuilder is enabled to use Azure Active Directory Authentication
-<!-- embedme ./src/samples/java/com/azure/communication/phonenumbers/ReadmeSamples.java#L52-L62 -->
+<!-- embedme ./src/samples/java/com/azure/communication/phonenumbers/ReadmeSamples.java#L54-L64 -->
```

Comment:
How to verify these line numbers are correct?

Reply:
`npx embedme README.md` populate the code, these snippets are not manually added.